lib/resty/mlcache/ipc.lua

```diff
@@ -137,14 +149,18 @@ function _M:broadcast(channel, data)
 end
 
 
-function _M:poll()
+function _M:poll(max_sleep)
```

Comment:
the `max_sleep` is more a generic setting imo, so better of in the initialization. Also because it saves some runtime checks and balances.